tests/test_common_features.py

```diff
@@ -15,10 +15,26 @@
 COMPONENT_PYTHON_BASE = 'python'
 COMPONENT_REACT_BASE = 'react'
 
+
+def bool_converter(s):
```

Comment:
We can instead check if the input parameter is equal to a valid boolean value (and if not, throw an `Exception`) -- there are many ways that someone can mistype `True` or `False` :)